connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/EmailCloudEventDataExtractor.java

```diff
@@ -51,5 +51,7 @@ public void extract(final Exchange exchange, final JsonObject cloudEventData) {
         exchange.setProperty(ExchangeProperty.SUBSCRIBERS, subscribers);
         exchange.setProperty(ExchangeProperty.UNSUBSCRIBERS, unsubscribers);
         exchange.setProperty(ExchangeProperty.EMAIL_RECIPIENTS, emails);
+        exchange.setProperty(ExchangeProperty.EMAIL_SENDER, cloudEventData.getString("email_sender"));
+        exchange.setProperty(ExchangeProperty.EMAIL_DEFAULT_RECIPIENT, cloudEventData.getString("email_default_recipient"));
```

Comment:
The email sender and default recipient may change depending on the app/bundle, but for a given notification we'll always use the same value for both. Therefore, we could only have one property here (EMAIL_SENDER) and pass its value in the two new fields from the request sent to BOP/MBOP.

Reply:
All right, changed!